package.json

```diff
@@ -9,9 +9,9 @@
   "scripts": {
     "lint": "standard",
     "lint:fix": "standard --fix",
-    "unit:template-ts-esm": "TS_NODE_PROJECT=./templates/app-ts-esm/tsconfig.json tap templates/app-ts-esm/test/**/*.test.ts --no-coverage --node-arg=--loader=ts-node/esm --timeout 100",
-    "unit:cli": "tap \"test/**/*.test.{js,ts}\" --no-coverage --timeout 200",
-    "unit:templates-without-ts-esm": "tap \"templates/app/**/*.test.js\" \"templates/app-esm/**/*.test.js\" \"templates/app-ts/**/*.test.ts\" --no-coverage --timeout 200",
+    "unit:template-ts-esm": " cross-env TS_NODE_PROJECT=./templates/app-ts-esm/tsconfig.json tap templates/app-ts-esm/test/**/*.test.ts --no-coverage --node-arg=--loader=ts-node/esm --timeout 100 --color -R specy",
```

Comment:
Tests were failing when an ENV var went into the test cycle, which doesn't work on windows.